src/components/ProgressForm/ProgressForm.tsx

```diff
@@ -73,7 +73,7 @@ const ProgressForm = ({ questions }: formProps) => {
     };
 
     return (
-        <form className={styles.form}>
+        <form className={styles.form} role="form">
```

Comment:
i don't think this was necessary.

Reply:
did you check if the task progress page was working after these changes?

Reply:
> did you check if the task progress page was working after these changes?

![Screenshot_20230531-080628.png](https://github.com/Real-Dev-Squad/website-status/assets/107541780/43add488-2305-4bd6-a13f-8444aa8290d7)
Is this the one your asking ?
i am unable to go to the route /progress/:id , do i need a special permission ?

Reply:
![Screenshot_20230531-122001](https://github.com/Real-Dev-Squad/website-status/assets/107541780/dbdf1c6c-5c10-4f99-a6a1-640afd01c202)
![Screenshot_20230531-121953](https://github.com/Real-Dev-Squad/website-status/assets/107541780/d29928ed-6cf1-4573-9396-a8173ff4509c)
![Screenshot_20230531-120924](https://github.com/Real-Dev-Squad/website-status/assets/107541780/d253ee3c-1991-4641-a7a6-3ced04b032c3)